packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1726,23 +1726,59 @@ Authenticators can be broadly classified into three kinds of factors. A factor r
 
-The number of authenticator class constraints in each constraint object be less than or equal to the value of `factorMode`. If the value is less, there are no constraints on any additional factors.
+The number of authenticator class constraints in each constraint object must be less than or equal to the value of `factorMode`. If the value is less, there are no constraints on any additional factors.
+
+| Property            | Data Type              | Description                                                                                                             | Supported Values                                                                                  |
+| -------------------- | ----------------- | ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| `types`              | Array  of Authenticator Types           | The Authenticator Types that are permitted                                                                           | [ `SECURITY_KEY`, `PHONE`, `EMAIL`, `PASSWORD`, `SECURITY_QUESTION`, `APP`, `FEDERATED` ]                         |
+| `methods`            | Array of Authenticator Methods           | The Authenticator Methods that are permitted                                                                          | [ `PASSWORD`, `SECURITY_QUESTION`, `SMS`, `VOICE`, `EMAIL`, `FIDO2`, `PUSH`, `SIGNED_NONCE`, `OTP`, `WEBAUTHN`, `DUO`, `IDP` ] |
+| `hardwareProtection` | String            | Indicates if any secrets or private keys that are used during authentication must be hardware protected and not exportable. This property is only set for `POSSESSION` constraints.| `REQUIRED`, `OPTIONAL`                                                                            |
+| `deviceBound` | String            | Indicates if device-bound factors are required. This property is only set for  `POSSESSION` constraints. | `REQUIRED`, `OPTIONAL`                                                                            |
+| `phishingResistant` | String            | Indicates if phishing-resistant factors are required. This property is only set for `POSSESSION` constraints. | `REQUIRED`, `OPTIONAL`                                                                            |
+| `userPresence` | String            | Indicates whether the user needs to approve a prompt in Okta Verify or provide biometrics (meets NIST AAL2 requirements). This property can be set in `POSSESSION` constraint only.| `REQUIRED`, `OPTIONAL`                                                                            |
+| `reauthenticateIn`   | String (ISO 8601) | The time that the End User must re-authenticate, regardless of user activity                            | N/A                                                                                               |
```

Comment:
This table should have a `Default` column, because `userPresence` defaults to `REQUIRED` and the others default to `OPTIONAL`.